lib/wallet/api/http/Cardano/Wallet/Api/Http/Shelley/Server.hs

```diff
@@ -3116,7 +3116,6 @@ balanceTransaction
         balancedTx <- liftHandler
             . fmap (Cardano.InAnyCardanoEra Write.cardanoEra . fst)
             $ Write.balanceTransaction
-                (MsgWallet . W.MsgBalanceTx >$< wrk ^. W.logger)
```

Comment:
IIRC the only place the tracer was used from — the `transactions-balance` endpoint, which itself rarely is used.